autoload/test/strategy.vim

```diff
@@ -6,7 +6,7 @@ function! test#strategy#basic(cmd) abort
   if has('nvim')
     -tabnew
     call termopen(a:cmd)
-    if !get(g:, 'test:basic:start_normal', 0)
+    if !get(g:, 'test#basic#start_normal', 0)
```

Comment:
`test:basic:start_normal` is not a valid variable name. And README mentions the valid variable name. [See](https://github.com/vim-test/vim-test/blob/dfbf93d71b63699d54accd37f12ba328b8ac311a/README.md?plain=1#L156).